pyproject.toml

```diff
@@ -60,4 +60,43 @@ target-version = "py38"
 # Ignore deleting imports in __init__.py files as they're actually being used.
 "__init__.py" = ["F401"]
 "**/settings/label_studio.py" = ["F405"]
-"test*.py" = ["F811"]
\ No newline at end of file
+"test*.py" = ["F811"]
+
```

Comment:
Not 100% sure there is a blue setting for import removal, but as it seemingly removed a few needed imports, maybe look into if there's a setting for disabling that